docs/introduction.md

```diff
@@ -68,17 +68,21 @@ npm install --save-dev @testing-library/react-hooks
 ### Peer Dependencies
 
 `react-hooks-testing-library` does not come bundled with a version of
-[`react`](https://www.npmjs.com/package/react) or
-[`react-test-renderer`](https://www.npmjs.com/package/react-test-renderer) to allow you to install
-the specific version you want to test against. Generally, the installed versions for `react` and
-`react-test-renderer` should have matching versions:
+[`react`](https://www.npmjs.com/package/react) to allow you to install the specific version you want
```

Comment:
I wonder if this could be made more concise, I think at least for now it gets the point across though.